src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/SymbolicRegexNode.cs

```diff
@@ -409,6 +409,21 @@ public bool IsDotStar
             }
         }
 
+        /// <summary>Returns true if this is equivalent to [\0-\xFFFF] </summary>
+        public bool IsDot
```

Comment:
This naming is a little confusing, since `.` maps to `[^\n]` unless RegexOptions.Singleline is provided.  This calls it "AnyClass":
https://github.com/dotnet/runtime/blob/c5e31e676b5f486741543c4e54f6b762cfae6c12/src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCharClass.cs#L73
Maybe we do the same here?

Reply:
Sounds good. I'll rename IsDot to IsAnyChar and IsDotStar to IsAnyStar